### ISSUE TITLE
[Bug] Integration going inresponsive after some time

### ISSUE THREAD

**Issue by scetu** [Vaskivskyi/ha-asusrouter#242]
### The problem

I have been using integration for few weeks, and noticed that it becomes unresponsive after some time (days) and Reloading integration from HA leads to errors in error log `ValueError: Config entry was never loaded!` I've spotted this mainly because device_tracker entities has not been updated.
I've tried to increase scan interval to 30 seconds from default 10 seconds, but it happened again.

### Your device model

RT-AX58U

### Firmware type

Merlin

### Firmware version

386.7_2

### Integration version with the issue

0.8.2

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

2022.10.3

### Is there anything useful in the logs?

```txt
Logger: homeassistant.config_entries
Source: helpers/entity_component.py:175
First occurred: 4:28:58 PM (15 occurrences)
Last logged: 5:05:31 PM

Error unloading entry 192.168.1.1 for binary_sensor
Error unloading entry 192.168.1.1 for device_tracker
Error unloading entry 192.168.1.1 for light
Error unloading entry 192.168.1.1 for sensor
Error unloading entry 192.168.1.1 for switch
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 498, in async_unload
    result = await component.async_unload_entry(hass, self)
  File "/usr/src/homeassistant/homeassistant/components/binary_sensor/__init__.py", line 175, in async_unload_entry
    return await component.async_unload_entry(entry)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 175, in async_unload_entry
    raise ValueError("Config entry was never loaded!")
ValueError: Config entry was never loaded!
```

### Diagnostics information

[config_entry-asusrouter-ce972f8c6868c630754e4532848895c1.json.txt](https://github.com/Vaskivskyi/ha-asusrouter/files/9778178/config_entry-asusrouter-ce972f8c6868c630754e4532848895c1.json.txt)

**Comment by Vaskivskyi** [comment]
Hello, @scetu,

I am sorry, that the integration does not work well for you.

The exception you see when trying to reload the integration is because it has already crashed and HA cannot "stop" it properly, before starting again.

Can you check what the log shows when sensors stop updating?

**Comment by Vaskivskyi** [comment]
I suppose that `Auto Logout` only matters when no actions are taken, but the integration sends requests regularly. Also, the integration behaves more like the Asus app for phones, rather than the usual web login.

At least, my AX88U has the default 30 minutes set and it was never a problem. But let's see when and how this will happen to your device next time

**Comment by Vaskivskyi** [comment]
Hello, @scetu,

Is this still with version 0.8.2 of the integration? Can you check whether this problem occurs with the new version 0.9.0 released today? There was a fix implemented that should help with the top error (where you have `Error doing job: Task exception was never retrieved`)

With the bottom one, in principle, HA should be able to handle and recover sensor updates. But we need to go step-by-step.

**Comment by neuroverflow** [comment]
Hi! I'm not gonna add any additional detail but I have the exact same issue and logs. Just to let you know it's not an isolated problem.
Thank you for your hard work!

Home Assistant 2022.10.4
Supervisor 2022.10.0
Operating System 9.2

**Comment by Vaskivskyi** [comment]
Hello,

Version 0.9.2 of the integration is already available through HACS. It should fix sensors being stuck after receiving errors. As a result, the integration should not be stuck anymore in a state, from which it cannot be reloaded (so should in principle fix that `Config entry was never loaded!` error). Can you try it out?

**Comment by Vaskivskyi** [comment]
Hey @scetu,

Thanks for the update on the issue! :+1: It's great, the problem is fixed now.

Yes, it does not prevent your device from answering with an error (and in this case, your sensor(s) won't update until the correct data is received), but now neither of the sensors should be stuck.

Since the issue seems to be resolved, I will close it. If you will experience any other problems with the integration, please, feel free to open a new issue.

**Comment by neuroverflow** [comment]
I was losing the sensors quite quickly, 4-6h. So far after a day it seems stable!
I can remove the integrated asus integration and use yours instead! Great job!

**Comment by Vaskivskyi** [comment]
Hey, @scetu,

Can you please check the new version 0.11.3? Unfortunately, there were a couple of bugs in 0.11.2, which are device-dependent and which I was not able to track with my only device.
